test/parallel/test-repl-reverse-search.js

```diff
@@ -342,7 +341,8 @@ function runTest() {
 
       if (opts.checkTotal) {
         assert.deepStrictEqual(lastChunks, expected);
-      } else if (expected.length !== 0) {
+      } else if (expected.length !== i) {
+        console.error(tests[numtests - tests.length - 1]);
```

Comment:
is the `console.error()` required?

Reply:
It is only triggered in an error case and helps debugging. It is otherwise difficult to identify where the error came from.